components/predict-form/Predict.tsx

```diff
@@ -359,6 +359,10 @@ export function Predict({
     }
   }
 
+  // used to calculate width of TagsSelect menu
+  const containerRef = useRef<HTMLDivElement>(null);
```

Comment:
nit: `;` should be removed by prettier

I use this as my cursor "Rules for AI", and find it doesn't generate semicolons: `Always use typescript, React, Tailwind, daisy-ui, clsx, HeroIcons, and TRPC. No semicolons. Prefer useState to React.useState. Use typed reducers (inline type). Prefer inline function props.`

Reply:
Thanks, clearly I also have my format on save setup wrong, I'll fix both